### PR TITLE
Tung tran

### PR DESCRIPTION
**Pull request by trthtn0508**
Check giùm nha.

### CODE REVIEW

File: firebase.json [hunk 1]
```diff
@@ -188,7 +188,8 @@
       { "source": "/go/restoring-anonymous-routes", "destination": "https://docs.google.com/document/d/1vnjDruoiDz0eEd4eVzg3IS6W_LRaFxNSlKWtU6iXp2M/edit?usp=sharing", "type": 301 },
       { "source": "/go/router-and-widgetsapp-integration", "destination": "https://docs.google.com/document/d/1QodTFmwVr1yk8dyQvoqkeILxvm92oikB1bVhHTtp6R4/edit?usp=sharing", "type": 301 },
       { "source": "/go/move-mouse-tracker-to-rendering", "destination": "https://docs.google.com/document/d/1dYNOTZ4kdq6ndNHopjUlrGlGBw3W-3jMbnoNJouIYtE/edit?usp=sharing", "type": 301 },
-      { "source": "/go/input-field-autofill", "destination": "https://docs.google.com/document/d/1wYLsoc7NiHl2jFueB4Ros09E3nDCotdWVovDCFSMcAk/edit?usp=sharing", "type": 301 }
+      { "source": "/go/input-field-autofill", "destination": "https://docs.google.com/document/d/1wYLsoc7NiHl2jFueB4Ros09E3nDCotdWVovDCFSMcAk/edit?usp=sharing", "type": 301 },
+      { "source": "/go/text-field-characters", "destination": "https://docs.google.com/document/d/1OOFW0PEZf0orBl445YXJ3nfjyKOqZdmeQ6KU3glUh7s/edit?usp=sharing", "type": 301 }
```
Comment (truongsinh) [inline firebase.json]: some weird unrelated to translation stuff?

File: src/docs/codelabs/index.md [hunk 2]
```diff
@@ -3,109 +3,154 @@ title: Codelabs
 description: "Codelabs help you quickly get started programming Flutter."
 ---
 
+The Flutter codelabs provide a guided,
+hands-on coding experience. Some codelabs
```
Comment (truongsinh) [inline src/docs/codelabs/index.md]: some weird unrelated to translation stuff?

**Review by truongsinh — CHANGES_REQUESTED** [review]
some weird unrelated to translation stuff?

**Comment by duynht** [comment]
@truongsinh should we close this and start a clean PR?

**Comment by truongsinh** [comment]
Sure, let's close and open a clean new one.

**Pull request closed without merging**